traffic_ops/app/lib/API/DeliveryServiceMatches.pm

```diff
@@ -28,11 +29,19 @@ sub index {
 	my $format = $self->param("format") || "";
 
 	my $rs;
-	if ( &is_privileged($self) ) {
+	# TO the reviewer: Do we need to override the "is_priviledged" here byt the standard "ignore_ds_user_table" flag?
+	# What is the reason of the is_priv test - was someone just dussmissed the ds_tmuser table tests
+	if ( &is_privileged($self)) {
```

Comment:
no idea why it was restricted to "is privileged" which is really the same is "is operations or above" since ldap only users don't work any more